CHANGELOG.md

```diff
@@ -92,6 +89,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1664**
   - Updated `dbIndexer` Lambda to remove hardcoded references to DynamoDB table names.
 
+- **CUMULUS-1698**
+  - Change variable `saml_launchpad_metadata_path` to `saml_launchpad_metadata_url` in the `tf-modules/cumulus` Terraform module.
```

Comment:
This line looks like it's duplicated above in the Breaking Changes.   Is that intentional?

Reply:
Yeah I think that is fine. I just didn't want a separate `Fixed` header up top so moved it.